src/internal/auth/interceptor.go

```diff
@@ -37,45 +38,42 @@ var authHandlers = map[string]authHandler{
 	"/auth.API/WhoAmI":       unauthenticated,
 	"/auth.API/GetOIDCLogin": unauthenticated,
 
-	// TODO: restrict GetClusterRoleBinding to cluster admins?
-	// TODO: split GetScope for self and others
-	// TODO: split GetAuthToken for self and others
-	// TODO: split RevokeAuthToken for self and others
 	// TODO: split GetGroups for self and others
-	"/auth.API/GetConfiguration":  authenticated,
+	// TODO: restrict GetClusterRoleBinding to cluster admins?
 	"/auth.API/CreateRoleBinding": authenticated,
 	"/auth.API/GetRoleBinding":    authenticated,
 	"/auth.API/ModifyRoleBinding": authenticated,
-	"/auth.API/DeleteRoleBinding": authenticated,
 	"/auth.API/RevokeAuthToken":   authenticated,
 	"/auth.API/GetGroups":         authenticated,
-	"/auth.API/GetAuthToken":      admin,
-	"/auth.API/SetConfiguration":  admin,
-	"/auth.API/ExtendAuthToken":   admin,
-	"/auth.API/SetGroupsForUser":  admin,
-	"/auth.API/ModifyMembers":     admin,
-	"/auth.API/GetUsers":          admin,
-	"/auth.API/ExtractAuthTokens": admin,
-	"/auth.API/RestoreAuthToken":  admin,
-	"/auth.API/Deactivate":        admin,
+	"/auth.API/GetConfiguration":  authenticated,
```

Comment:
There's a permission defined for `AUTH_GET_CONFIG`, but it looks like this doesn't use it—is it for futureproofing or would it maybe make sense to get rid of it?

Reply:
Nevermind, just saw https://github.com/pachyderm/pachyderm/pull/5785, which addresses this. Marking resolved